Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/RNWServiceTest.cs

```diff
@@ -113,5 +120,116 @@ public void WhenPostValidRequestWithException_ThenReturnException()
             Assert.ThrowsAsync(Is.TypeOf<Exception>(),
                                async delegate { await _rnwService.CreateNewRadioNavigationWarningsRecord(_fakeRadioNavigationalWarning, CorrelationId); });
         }
+
+        [Test]
+        public async Task WhenCallGetRadioNavigationWarnings_ThenReturnListAsync()
+        {
+            A.CallTo(() => _fakeRnwRepository.GetRadioNavigationWarningsAdminList()).Returns(GetFakeRadioNavigationalWarningList());
+            RadioNavigationalWarningsAdminListFilter result = await _rnwService.GetRadioNavigationWarningsForAdmin(1, null, null, string.Empty);
+            Assert.IsTrue(result.RadioNavigationalWarningsAdminList.Count > 0);
+            Assert.IsTrue(result.PageCount == 2);
+            Assert.IsTrue(result.SrNo == 0);
+            Assert.IsTrue(result.CurrentPageIndex == 1);
```

Comment:
Tests should be using `AreEqual`, `Greater` etc. as appropriate rather that always `IsTrue` please:
```suggestion
            Assert.Greater(result.RadioNavigationalWarningsAdminList.Count, 0);
            Assert.AreEqual(2, result.PageCount);
            Assert.AreEqual(0, result.SrNo);
            Assert.AreEqual(1, result.CurrentPageIndex);
```

Same for following tests.

Reply:
Thanks, Changes done as requested.